code-challenges.js

```diff
@@ -33,19 +68,51 @@ var myForNumbers2 = [2, 5, -8, 20, 32]
 var myMapNumbers1 = [87, 6, 28, 16, 2]
 var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+//***psudo***
+//function that takes an array
+//create a variable to hold new array
+//use map to multiply each number by 5
+
+// const timesFiveMap = (array) => {
+//   let mappedFive = array.map(value => {
+//     return value * 5
+//   })
+//   return mappedFive
+// }
 
+// **refactor without variable**
 
+const timesFiveMap = (array) => {
+  return array.map(value => {
+    return value * 5
+  })
+}
 
+console.log('Q3:',timesFiveMap(myMapNumbers1))
+console.log('Q3:',timesFiveMap(myMapNumbers2))
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+// ***psudo*** 
+// function that takes a string 
+// make string an array 
+// iterate though new array
+// filter out vowels 
+// return array as a string without vowels
 
+const onlyVowels = (str) => {
```

Comment:
```suggestion
const removeVowels = (str) => {
```
generally function naming conventions start with verbs to indicate intent